lib/utils/typeGuards.js

```diff
@@ -39,6 +39,6 @@ module.exports.isComment = function isComment(node) {
  * @param {Node} node
  * @returns {node is (Node & {source: NodeSource})}
  */
-module.exports.hasSource = function isComment(node) {
-	return !!node.source;
+module.exports.hasSource = function hasSource(node) {
```

Comment:
We could remove the function name here; not sure what's used more in the codebase. This change that ensures consistency.